ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/legaladvisor/CCDHearingTimeType.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.cmc.ccd.domain.legaladvisor;
+
+public enum CCDHearingTimeType {
+    THIRTY_MIN,
+    ONE_HOUR,
+    ONE_HALF_HOUR,
```

Comment:
We can rename this better

Reply:
`CCDHearingDurationType`?

Reply:
That sounds better. Also we can rename `ONE_AND_HALF_HOUR`.